### PR TITLE
Python 3 fixes to Tribler/Core/Utilities

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: Tribler/Core/Utilities/utilities.py [hunk 1]
```diff
@@ -487,4 +487,4 @@ def translate_peers_into_health(peer_info_dicts):
 
 def unichar_string(text):
     """ Unicode character interpretation of text for Python 2.7 """
-    return u''.join(unichr(ord(t)) for t in text)
+    return u''.join(six.unichr(ord(t)) for t in text)
```
Comment (qstokkink) [inline Tribler/Core/Utilities/utilities.py]: We can replace this method entirely with this: https://github.com/Tribler/tribler/pull/4094#discussion_r240111555

Comment (cclauss) [reply Tribler/Core/Utilities/utilities.py]: I am unclear what you mean.

Comment (qstokkink) [reply Tribler/Core/Utilities/utilities.py]: Well I guess it's not really relevant to your changes. But this function does exactly what I linked in that comment. At some point (probably not in this PR), we should replace our custom stuff with builtin library functionality.

**Comment by devos50** [comment]
@cclauss could you please rebase this PR (and your other open PRs) on the latest commit on `devel`? We made various fixes to the unit tests 👍

**Pull request merged**